docs/api_index.rst

```diff
@@ -30,6 +30,14 @@ insights.core.dr
     :members:
     :exclude-members: requires, optional, metadata, group, tags
 
+insights.core.exceptions
+---------------------
```

Comment:
nitpick :), you may want to adjust the length of this line.

Reply:
Yeah it turns out to be the reason it's not building, I'll correct it thanks.